projects/plugins/boost/app/features/optimizations/cloud-css/Cloud_CSS.php

```diff
@@ -213,4 +214,28 @@ public function add_critical_css_constants( $constants ) {
 
 		return $constants;
 	}
+
+	/**
+	 * Return whether the Cloud CSS has an error or not.
+	 *
+	 * @return boolean
+	 */
+	public function has_health_problem() {
+		$cloud_css = new Critical_CSS_State( 'cloud' );
+
+		return $cloud_css->get_status() === 'error';
+	}
+
+	/**
+	 * Updates the total problem count for Boost if something's
+	 * wrong with Cloud CSS.
+	 *
+	 * @param  int $count The current problem count.
+	 * @return int
+	 */
+	public function update_total_problem_count( $count ) {
```

Comment:
Could this method be missing a check whether $count is an `int`?

Reply:
That's a good point - thanks!